.github/workflows/CI.yaml

```diff
@@ -75,3 +75,25 @@ jobs:
         file: ./coverage.xml
         flags: unittests
         name: codecov-${{ matrix.os }}-py${{ matrix.python-version }}
+  
+  test_pypi:
```

Comment:
probably don't want to call it pypi (since that'd refer to the pypi repository?)